Cargo.toml

```diff
@@ -40,6 +40,7 @@ serde_json = "1.0.61"
 shellexpand = "2.1.0"
 termcolor = "1.1"
 terminal_size = "0.1.15"
+thiserror = "1.0.30"
```

Comment:
Why not using the existing anyhow? I'm not so fan to have 2 different error management tools in the same codebase, let's stay consitant.

Reply:
I wanted to use [`thiserror`](https://crates.io/crates/thiserror) to create my own custom errors + messages for the wizard. But since it looks like that I don't need to write our own wizard, due to your links, I think that I'll remove this dependencie then :)